app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -94,7 +94,7 @@ def lighthouse_facilities_service
     end
 
     def lighthouse_facilities_params
-      params.permit(
+      params.except(:format).permit(
```

Comment:
What is this for?

Reply:
I was getting a warning in the console that this non-permitted param was present, and added this to clear that out.